src/CodeCompliance/Infrastructure/Adapter/CodeBaseService.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace CodeCompliance\Infrastructure\Adapter;
```

Comment:
Are you sure that we have correct namespace?

Reply:
Thanks a lot, renamed to service